### DISCUSSION TITLE
Update Issues from 2.3.21 - 2.3.3

### DISCUSSION THREAD

**Discussion by kjsdjrmf** [Security-Onion-Solutions/securityonion#3418] [Unsupported Versions]
Is anyone else having issues updating from version 2.3.21 - 2.3.3 via soup?

Seems to fail when trying to import the KEYS

"Executing command with retry support: curl -sSL https://raw.githubusercontent.com/Security-Onion-Solutions/securityonion/master/KEYS -o /root/sosigs/KEYS
curl: (35) TCP connection reset by peer
Results:  (35)
Command failed with exit code 35; will retry in 10 seconds (1 / 50)..."

**Comment by dougburks** [comment]
Does your box have Internet access?

[a comment by an automated account is not shown]
